apps/common/test/lexical/ast/detection/struct_field_value_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Lexical.Ast.Detection.StructFieldValueTest do
+  alias Lexical.Ast.Detection
+
+  use Lexical.Test.DetectionCase,
+    for: Detection.StructFieldValue,
+    assertions: [[:struct_field_value, :*]],
+    skip: [
+      [:struct_fields, :*],
+      [:struct_reference, :*],
+      [:struct_field_key, :*]
```

Comment:
Aren't key and value mutually exclusive? I don't understand why `field_key` is skipped here.

Reply:
because values can appear inside the tests for keys, and the tests will fail because of this. 
Keep in mind, those tests show where keys can appear, and if anything else appears in the test, they will fail.